SingularityUI/app/styles/colors.styl

```diff
@@ -2,7 +2,7 @@ $linkBlue = #08c
 
 $beige = #f3efe0
 $grey = #444444
-
+$gray-light = #dadada
```

Comment:
For the sake of consistency maybe use the [correct](http://en.wikipedia.org/wiki/Grey) (joke (not really (well, maybe a little))) spelling here.